.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - "6"
```

Comment:
@wcjordan 
Are we upgrading the node versions just because of the Travis CI errors found in v1.0-beta, [here](https://travis-ci.org/schrodinger/fixed-data-table-2/jobs/477152906) and [here ](https://travis-ci.org/schrodinger/fixed-data-table-2/jobs/477152905)(due to commit 3713245)?

> error har-validator@5.1.3: The engine "node" is incompatible with this module. Expected version ">=6".

Can you guide me on what this means? I'm guessing `har-validator` is a dependency for FDT which does not work on node.js versions less than 6?
But I couldn't find `har-validator` on `package.json` and `yarn.lock` though, so not sure how this comes into play.

Reply:
That's the right direction.  There are a couple things I can help clarify.

You are correct, this file controls the environment we run our tests in on Travis CI.  Previously it was set to nodejs 4, 5, & 6.  As you pointed out, this is now failing on v1.0-beta due to the recent updates to our dependencies due to har-validator not working on node 4 & 5.  

Har-validator doesn't appear in package.json because it is a transitive dependency (we don't rely on it directly, but less and jsdom do).  You can find it [referenced in yarn.lock](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/yarn.lock#L2819).

The issue here is we shouldn't be relying on node 4, 5, & 6.  They're really old (in fact 8 is nearly end of life).  You can learn more about nodejs release lifecycle here: https://github.com/nodejs/Release

Let me know if you have any additional questions.

Reply:
Thanks for the explanation.